components/org.wso2.carbon.identity.oauth.endpoint/pom.xml

```diff
@@ -296,7 +301,7 @@
                                         <limit implementation="org.jacoco.report.check.Limit">
                                             <counter>COMPLEXITY</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.55</minimum>
+                                            <minimum>0.53</minimum>
```

Comment:
Ensure we add more tests @ChitharaKarunasekera 
As @chamathns suggested hope this is fine for the time being for expediting  6.2.0-alpha

Reply:
+1

Reply:
let's make sure we revert this change with the test PR

Reply:
+1.